packages/fxa-payments-server/Gruntfile.js

```diff
@@ -4,11 +4,7 @@
 
 module.exports = function (grunt) {
   const srcPaths = [
-    // 'src/branding.ftl' is temporary
-    // and will be replaced with '../fxa-shared/lib/l10n/branding.ftl'
-    // in a later ticket - will require coordination with l10n to resolve
-    // conflicting IDs for identical terms.
-    'src/branding.ftl',
+    '../fxa-shared/l10n/branding.ftl',
```

Comment:
```suggestion
    '../fxa-shared/l10n/branding.ftl',
```

Also - we might need to double check this. I think this might result in the `branding.ftl` file getting extracted twice.

Reply:
@bcolsson is there an easy way to double check this on our side? This is now in alignment with the way `fxa-settings` works, so if it is working there, I would think it would work here too.

Reply:
@dschom - I'm not an expert at the build system at all, but I recall this happening in different PRs where we've had to revert `branding.ftl` getting added to the srcPaths (#15527).

Looking at `fxa-settings` the `branding.ftl` file is being added to `testPaths` not `srcPaths`.

Reply:
I'll update to match `fxa-settings`. Thank you for the catch.

Reply:
@dschom @bcolsson updated. Please let me know if this is correct.